lib/class-wp-duotone-gutenberg.php

```diff
@@ -747,7 +747,7 @@ private static function enqueue_global_styles_preset( $filter_id, $duotone_selec
 	 */
 	public static function register_duotone_support( $block_type ) {
 		$has_duotone_support = false;
-		if ( property_exists( $block_type, 'supports' ) ) {
+		if ( $block_type instanceof WP_Block_Type ) {
```

Comment:
This line doesn't seem to appear at all in WP Core 

https://github.com/WordPress/wordpress-develop/blob/e83f5a1a602b86dbecbf8ee58d6b8761d53b6197/src/wp-includes/class-wp-duotone.php#L926

Reply:
At a glance, it appears that the code was refactored a little in core. The check for the block type is [handled](https://github.com/WordPress/wordpress-develop/blob/e83f5a1a602b86dbecbf8ee58d6b8761d53b6197/src/wp-includes/blocks.php#L1649) via the [call](https://github.com/WordPress/wordpress-develop/blob/e83f5a1a602b86dbecbf8ee58d6b8761d53b6197/src/wp-includes/class-wp-duotone.php#L931) to `block_has_support`.

Reply:
So is there a missing backport on this PR. Should we consider it done. It's very unfortunate that the code between core and Gutenberg for block supports have diverged this much.

We should either do:

 - Have the exact same code and use a package (like the block-library) sync
 - Remove the block supports from Gutenberg (consider that they're now maintained on Core)

It's probably something to do post release though.

Reply:
So to confirm, no further action on this and it can be marked as backported?

> Should we consider it done.

Can I check whether you intended to write `We should consider it done`?